src/components/ContentListTable/ContentListFilters.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement, useContext } from 'react';
+import { FlexItem, InputGroup, SelectVariant, TextInput } from '@patternfly/react-core';
+import DropdownSelect from '../DropdownSelect/DropdownSelect';
+import { FilterIcon } from '@patternfly/react-icons';
+import { ContentListContext } from './ContentListContext';
+
+const ContentListFilters = () => {
+    const filters= ['Name/URL', 'Version', 'Architecture'];
+    const versions = {};
+    const arches = {};
+
+    const {
+        repoParams,
+        setSearchQuery,
+        filterType,
+        setFilterType,
+        selectedVersions,
+        setSelectedVersions,
+        selectedArches,
+        setSelectedArches,
+    } = useContext(ContentListContext)
+
+    repoParams['distribution_arches'].forEach(arch => arches[arch.name] = arch.label );
```

Comment:
you'll want to to throw this into a useEffect with dependencies preventing it running every render.